airbyte-cdk/python/unit_tests/sources/declarative/test_manifest_declarative_source.py

```diff
@@ -206,7 +207,7 @@ def test_manifest_with_spec(self):
         source = ManifestDeclarativeSource(source_config=manifest)
         connector_specification = source.spec(logger)
         assert connector_specification is not None
-        assert connector_specification.documentationUrl == "https://airbyte.com/#yaml-from-manifest"
+        assert connector_specification.documentationUrl == AnyUrl("https://airbyte.com/#yaml-from-manifest")
```

Comment:
Test validation assertion changes: see [Url and Dsn types in pydantic.networks no longer inherit from str](https://docs.pydantic.dev/latest/migration/#url-and-dsn-types-in-pydanticnetworks-no-longer-inherit-from-str)